anilist.py

```diff
@@ -1015,14 +911,15 @@ def match_series_with_seasons(
                         % (plex_title)
                     )
         else:
+            #ignore the Plex year since Plex does not have years for seasons
```

Comment:
Sorry, I forgot a space here. Should be `# ignore the Plex year since Plex does not have years for seasons`

Reply:
No problem, fixed now :)